### ISSUE TITLE
Need some code for example

### ISSUE THREAD

**Issue by Parimal1238** [H1R0GH057/Anonymous#34]
I am just a beginniner in this world, need few code example for bruteforce attack

**Comment by Lamf4o** [comment]
I don't think you are asking that for learning purposes, I think you are asking for Brute-force scripts simply because you want to do an attack on someone. Am I correct?

**Comment by alpfapapa** [comment]
i attack only porn sites

**Comment by alpfapapa** [comment]
fuck pedophiles
